hw3/react-hw3/thrones/src/Components/Search/SearchComponent.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+
+import InputGroup from "react-bootstrap/InputGroup";
+import axios from "axios";
+import { Card, Row, Col } from 'react-bootstrap';
+
+export class SearchComponent extends Component {
```

Comment:
Throughout the React exercise a lot of classes were used, which is a very valuable thing to understand. However, it is more common to use functional components these days as they are a lot easier to work with. Was there a specific reason class components were used?